drivers/hmis_external_apis/public/schemas/referral.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "open-path-schemas/referral.json",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "additionalProperties": false,
+  "title": "Referral",
+  "description": "Notify HMIS that a client and household members are eligible for a posting",
+  "type": "object",
+  "required": ["referral_id", "referral_date", "service_coordinator", "posting_id", "program_id"],
+  "properties": {
+    "referral_id": { "type": ["integer", "string"] },
+    "referral_date": { "type": "string", "format": "date" },
+    "service_coordinator": { "type": "string" },
+    "posting_id": { "type": ["integer", "string"] },
+    "referral_request_id": { "type": ["integer", "string"] },
+    "program_id": { "type": ["integer", "string"] },
+    "unit_type_id": { "type": ["integer", "string"] },
+    "household_members": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["mci_id", "relationship_to_hoh", "first_name", "last_name", "dob"],
```

Comment:
This is correct, as listed here https://docs.google.com/spreadsheets/d/12wRLTjNdcs7A_1lHwkLUoKz1YWYkfaQs/edit#gid=1460281785 thanks!